Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs

```diff
@@ -235,22 +256,23 @@ private float GetFarthestPushDistanceAlongButtonAxis()
             Debug.Assert(initialTransform != null);
 
             float farthestDistance = 0.0f;
-            foreach (Vector3 p in touchPoints)
+
+            foreach (var touchEntry in touchPoints)
             {
-                float testDistance = GetProjectedDistance(initialTransform.position, WorldSpacePressDirection, p);
-                farthestDistance = Mathf.Max(farthestDistance, testDistance);
+                float testDistance = GetProjectedDistance(initialTransform.position, WorldSpacePressDirection, touchEntry.Value);
+                farthestDistance = Mathf.Max(testDistance, farthestDistance);
             }
 
             return Mathf.Clamp(farthestDistance, 0.0f, maxPushDistance);
         }
 
-        private void UpdatePressedState(float pushDistance, float previousPushDistance)
+        private void UpdatePressedState(float pushDistance)
         {
             // If we aren't in a press and can't start a simple one.
             if (!isPressing)
             {
                 // Compare to our previous push depth. Use previous push distance to handle back-presses.
```

Comment:
Might want to update the comment if we are no longer using the previousPushDistance